projProject/project.cpp

```diff
@@ -60,37 +210,76 @@ void filledInCircle(int x, int y, GLfloat radius) {
     glEnd();
 }
 
+//Draws frame for windows
+void drawCircle(float xC, float yC, int radius)
+{
+	const float pi = 3.141592653f;			// constant PI value
+	for (float angle = 0; angle <= 2 * pi; angle += 0.01)	{
+		glBegin(GL_POINTS);					// uses points to draw face
+			glVertex2f(xC + (radius*cos(angle)), yC + (radius*sin(angle)));				// draw the vertex point at the specified co-ordinate
+		glEnd();
+	}
+}
+
 // Draws 2 windows
 void drawWindows() {
-    glColor3f(.2, .4, 0);
-    filledInCircle(-100, -75, 20);
-    filledInCircle(100, -75, 20);
-}
+	//draws window frame
+	glPointSize(3);
+	glColor3f(0, 0, 0);
+	drawCircle(-100, -15, 50);
+	drawCircle(100, -15, 50);
 
-// Draws a house
-void drawHouse() {
-    glColor3f(0, .2, .4);
-    glBegin(GL_POLYGON);
-    glVertex2i(-200, -300);         // Bottom left
-    glVertex2i(-200, 0);           // Top left
-    glVertex2i(200, 0);            // Top right
-    glVertex2i(200, -300);          // Bottom right
-    glEnd();
+	//draws windows glass
+	glColor3f(0.439, 0.502, 0.565);		//slate grey
+	filledInCircle(-100, -15, 48);
+	filledInCircle(100, -15, 48);
+
+	glLineWidth(2.5);
+	// draws window pane
+	glColor3f(0, 0, 0); // color black
+	glBegin(GL_LINE_STRIP);
+	glVertex2i(-148,-15);
+	glVertex2i(-52, -15);
+	glEnd();
+	/*glBegin(GL_LINE_STRIP);
```

Comment:
If you don't want the code in just remove. Since we are using version control commenting out is not needed.

Reply:
I kept it for modification purpose, i will remove this before i merge it to master